testnet/stacks-node/src/chain_data.rs

```diff
@@ -362,10 +363,14 @@ impl MinerStats {
 
         for (_, commit) in active_miners_and_commits.iter() {
             let addr = commit.apparent_sender.to_string();
-            if dist.contains_key(&addr) {
-                continue;
+            match dist.entry(addr.to_owned()) {
+                Entry::Occupied(_) => {
+                    continue;
```

Comment:
`continue` isn't necessary because we're at the end of the loop, how about changing this block to an `if let Entry::Vacant(e)...`